### ISSUE TITLE
Paginator is not shown in Import Modules

### ISSUE THREAD

**Issue by tgaertner** [xperseguers/t3ext-ig_ldap_sso_auth#114]
Tested TYPO3 8.7.32 and TYPO3 9.5.22
As a result its not possible to manually import users that are located on following pages.

**Comment by xperseguers** [comment]
Actually the real issue here is that the paginator is really missing altogether! This is not a bug but a missing feature since the module was never built with this in mind. I agree that it would be really useful, just like being able to dynamically search in the table.

**Comment by tgaertner** [comment]
Mh the main Problem is that not all users are listed. Is there a limit? So if there are many users its kind of broken. Because the list is cut off somewhere I thought there might exist a paginator.

**Comment by xperseguers** [comment]
Yes there's a limit, it's hardcoded to 2000 iirc, just have a look at the source code, it's explained the reason of the limit. Originally, the import module was not designed for managing loads of users but just to make it easy to manually import few of them. Before that module, the extension was basically designed to import users on the fly while authenticating or importing them all with the corresponding scheduler task. So the import module was never thought as the ultimate way to import "some" users, it is supposed that you import all of them and if you want to filter them, then you are supposed to better prepare your LDAP connection with additional filter for groups and/or conditions on the DN.

**Comment by klodeckl** [comment]
I have the same problem (I always thougth the problem is the used ldap server until I read now that further pages are not implemented). I “fixed“ it by changing Classes/Utility/LdapUtility.php:
const MAX_ENTRIES = 50000;
and
const PAGE_SIZE = 10000;

This way all users get listed.

I would love to see the feature to use page 2, 3 etc.! :)
